src/modules/new-request-form/NewRequestForm.tsx

```diff
@@ -117,15 +117,8 @@ export function NewRequestForm({
                 onChange={(value) => handleChange(field, value)}
               />
             );
-          case "priority":
           case "organization_id":
-            return (
-              <DropDown
-                key={field.name}
-                field={field}
-                onChange={(value) => handleChange(field, value)}
-              />
-            );
+          case "priority":
           case "tickettype":
             return (
               <DropDown
```

Comment:
Curious, would it be possible to use `Tagger` here instead?

Reply:
In theory, it should be possible to use the same component for all the dropdowns.
Normal dropdowns don't support nesting and they don't require the complex logic implemented in the `Tagger` component, both for creating the nested options and passing the initial options. It is not a big deal to use the same component for both, they will just have root options and some unuseful code for the initial options.

There is also another caveat at the moment, since for the organizations dropdown we have a number as a value instead of a string for the options, and it requires us to convert them to string beforehand, so the `Tagger` component doesn't work.

Let's keep the two different components for now and eventually revisit them later when we improve the data passed from Curlybars.